### PR TITLE
Onboarding

### PR DESCRIPTION
**Pull request by ArtSp**
Onboarding:

- Dynamic font size ( Onboarding related views )
- Scroll view for welcome vc 
- Exposure switch redesign
- UI updates for links.

### CODE REVIEW

File: Podfile.lock [hunk 1]
```diff
@@ -123,4 +123,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 69fe77633cdb363bbf101efe31c9b95b5679465d
 
-COCOAPODS: 1.9.3
+COCOAPODS: 1.9.1
```
Comment (ULazdins) [inline Podfile.lock]: Downgrade ;(

File: apturicovid/welcome/WelcomeVC.swift [hunk 2]
```diff
@@ -16,7 +16,7 @@ class WelcomeVC: BaseViewController {
     let langViews = Language.allCases.map{ LanguageView.create($0) }
     
     //MARK: Text attributes:
-    let linkFont = UIFont.systemFont(ofSize: 14, weight: .medium)
+    var linkFont:UIFont { UIFont.preferredFont(forTextStyle: .body)}
```
Comment (ULazdins) [inline apturicovid/welcome/WelcomeVC.swift:19]: why var? Is it being mutated somewhere?

Also - I'd prefer getter body to be on a new line like
```
var linkFont:UIFont { 
    UIFont.preferredFont(forTextStyle: .body)
}
```

**Review by ULazdins — APPROVED** [review]

File: apturicovid/welcome/WelcomeVC.swift (hunk 2 above)
Comment (ArtSp) [reply apturicovid/welcome/WelcomeVC.swift:19]: It’s a computed property.

File: Podfile.lock (hunk 1 above)
Comment (ArtSp) [reply Podfile.lock]: It depends on your cocoa pods version. Doesn’t affect project. Will update version later

Comment (mzibara) [reply Podfile.lock]: I think you can upgrade your local version for consistency :)

**Review by mzibara — APPROVED** [review]

File: Podfile.lock (hunk 1 above)
Comment (ArtSp) [reply Podfile.lock]: Updated to 1.9.3

**Pull request merged**